simple_parsing/wrappers/dataclass_wrapper.py

```diff
@@ -77,7 +80,23 @@ def __init__(
                 child_wrapper.required = False
                 child_wrapper.optional = True
                 self._children.append(child_wrapper)
-
+            elif utils.is_final(field.type):
+                # Final fields have no business being set by the command line
+                continue
+            elif utils.is_literal(field.type):
+                choices = field.type.__args__
```

Comment:
Does this check for an `__args__` attribute work in all python versions (3.6+?)
I think it might be preferable to use a function for this if not, something like `get_type_arguments` (that I think I've already written somewhere in utils), or maybe a better function from `typing_extensions`.

Reply:
Thanks, I just found that function while adding support for lists.
Well, the tests did pass with all versions, so it seems to be good, but switching to the function (which does the same basically) is a good idea anyway. :ok_hand: